src/main/thrift/parquet.thrift

```diff
@@ -962,15 +967,19 @@ union ColumnOrder {
    *   BYTE_ARRAY - unsigned byte-wise comparison
    *   FIXED_LEN_BYTE_ARRAY - unsigned byte-wise comparison
    *
-   * (*) Because the sorting order is not specified properly for floating
-   *     point values (relations vs. total ordering) the following
+   * (*) Because the precise sorting order is ambiguous for floating
+   *     point types due to underspecified handling of NaN and -0/+0,
+   *     it is recommended that writers use IEEE_754_TOTAL_ORDER
+   *     for these types.
+   *
+   *     If TYPE_ORDER is used for floating point types, then the following
```

Comment:
This line threw me (at least while using my phone 😉...on my computer I can see `TYPE_ORDER` below). Maybe this could instead say "If this ordering is used for floating..." or "If this type-defined ordering..."